src/hooks/useSentEmailAmount.ts

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react';
+import { fetchEmailAmount } from '../components/request';
+
+const useSentEmailAmount = (): { sentEmailsNumber: number } => {
```

Comment:
- rename everything in this sense: useWeb3mailStats
- hooks need to be put inside the modules